servers/rendering/renderer_rd/renderer_canvas_render_rd.cpp

```diff
@@ -398,7 +396,34 @@ void RendererCanvasRenderRD::_bind_canvas_texture(RD::DrawListID p_draw_list, RI
 	r_last_texture = p_texture;
 }
 
-void RendererCanvasRenderRD::_render_item(RD::DrawListID p_draw_list, RID p_render_target, const Item *p_item, RD::FramebufferFormatID p_framebuffer_format, const Transform2D &p_canvas_transform_inverse, Item *&current_clip, Light *p_lights, PipelineVariants *p_pipeline_variants, bool &r_sdf_used) {
+void RendererCanvasRenderRD::_bind_canvas_texture(RD::DrawListID p_draw_list, RID p_texture, RS::CanvasItemTextureFilter p_base_filter, RS::CanvasItemTextureRepeat p_base_repeat, RID &r_last_texture, const PushConstant &push_constant) {
+	if (p_texture == RID()) {
+		p_texture = default_canvas_texture;
+	}
+
+	if (r_last_texture == p_texture) {
+		return; //nothing to do, its the same
```

Comment:
```suggestion
		return; // Nothing to do, its the same.
```
[Comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide) nitpick, applies to all modified code even if it is just copied or relocated

Reply:
Also, "its" -> "it's"